src/u2f.c

```diff
@@ -39,6 +41,9 @@
 #define APPID_BOGUS_CHROMIUM "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
 #define APPID_BOGUS_FIREFOX "\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0"
 
+#define U2F_ASSERT(cond, err) do { if (!(cond)) { Abort("U2F error: " err); } } while(0);
+#define U2F_USB_SEND(a, b) do { if (usb_send_packet(a, b) != QUEUE_ERR_NONE) {Abort("DAMMIT!");} } while(0);
```

Comment:
better abort message please, can't have sth like this pop up on production devices :smile: 

please use functions over macros

Reply:
Old code, was refactored.